pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -98,6 +97,52 @@ func Execute(configFile io.Reader) {
 		pruneAccessRecords,
 	)
 
+	// Registry extensions endpoint provides extra functionality to handle the image
+	// signatures.
+	extensionsRouter := app.NewRoute().PathPrefix("/extensions/v2/").Subrouter()
+	var (
+		getSignatureAccess = func(r *http.Request) []auth.Access {
+			return []auth.Access{
```

Comment:
This can be empty. The repository check (`repository:namespace/name:get`) is [done anyway by upstream dispatcher](https://github.com/docker/distribution/blob/master/registry/handlers/app.go#L798).

Unless we want to do the same check twice.